### PR TITLE
Create hjgaming.json

### PR DESCRIPTION
**Pull request by Hjgaming**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [ ] The file is in the `domains` folder and is in the JSON format.
- [ ] The file's name is all lowercased and alphanumeric. 
- [ ] You have completed your website. 
- [ ] The website is reachable.  
- [ ] The CNAME record doesn't contain `https://` or `/`.  
- [ ] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/hjgaming.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "discord connected domain",
+  "repo": "https://github.com/Hjgaming/register-1",
+  "owner": {
+    "username": "hjgaming",
+    "email": "jariwalaharshal80@gmail.com"
+  },
+  "record": {
+    "TXT": ["dh=234fc226b4f9dab9135d3a91e77d623924640e81"]
```
Comment (CuteDog5695) [inline domains/hjgaming.json:9]: This is not supposed to be for the main file. Please read our documentation.

**Review by iostpa — CHANGES_REQUESTED** [review]

**Review by Hjgaming — COMMENTED** [review]
new added pls re review

File: domains/hjgaming.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "discord connected domain",
+  "repo": "https://github.com/Hjgaming/register-1",
+  "owner": {
+    "username": "hjgaming",
+    "email": "jariwalaharshal80@gmail.com"
+  },
+  "record": {
+    "TXT": ["vc-domain-verify=hjgaming.is-a.dev,b1a85aa1f9f16d27c005"]
```
Comment (DEV-DIBSTER) [inline domains/hjgaming.json:9]: Vercel verification records should be under a file called _vercel.domain.json, this file should be the actual A or CNAME records.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

**Review by DEV-DIBSTER — APPROVED** [review]

File: domains/_vercel.hjgaming.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
+{
+    "description": "discord connected domain",
+    "repo": "https://github.com/Hjgaming/register-1",
+    "owner": {
+        "username": "hjgaming",
+        "email": "jariwalaharshal80@gmail.com"
+    },
+    "record": {
+        "TXT": [
+            "vc-domain-verify=hjgaming.is-a.dev,b1a85aa1f9f16d27c005"
```
Comment (DEV-DIBSTER) [inline domains/_vercel.hjgaming.json:10]: You also need to create the root file as well.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

**Comment by Hjgaming** [comment]
close

**Pull request closed without merging**